bundle/Templating/Twig/IbexaAdminUIRuntime.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\BetterIbexaAdminUIBundle\Templating\Twig;
+
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException;
+use Netgen\IbexaSiteApi\API\FilterService;
+use Twig\Extension\RuntimeExtensionInterface;
+
+class IbexaAdminUIRuntime implements RuntimeExtensionInterface
+{
+    private FilterService $filterService;
+
+    /**
+     * @param \Netgen\IbexaSiteApi\API\FilterService $filterService
```

Comment:
PHPDoc is superfluous.

Reply:
fixed in b2d5f51b27ccdcfde4f08871269de875d54f1d44